src/PostgREST/Network.hs

```diff
@@ -13,5 +13,5 @@ resolveHost sock = do
   sn <- NS.getSocketName sock
   case sn of
     NS.SockAddrInet _ hostAddr ->  pure $ Just $ fromString $ show $ fromHostAddress hostAddr
-    NS.SockAddrInet6 _ _ hostAddr6 _ -> pure $ Just $ fromString $ show $ fromHostAddress6 hostAddr6
+    NS.SockAddrInet6 _ _ hostAddr6 _ -> pure $ Just $ fromString $ "[" ++ show (fromHostAddress6 hostAddr6) ++ "]"
```

Comment:
>  [::1]:80

@taimoorzaeem Can you elaborate why the above is the right format? Does it follow an RFC? If so can we add the reference in a comment?

Reply:
Yes, sure. FYI, this is in [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986#section-3.2.2). Let me add a comment.

Reply:
Aha, I see it now:
> A host identified by an Internet Protocol literal address, version 6
   [[RFC3513](https://datatracker.ietf.org/doc/html/rfc3513)] or later, is distinguished by enclosing the IP literal
   within square brackets ("[" and "]")